torch/csrc/jit/attributes.h

```diff
@@ -195,7 +201,7 @@ struct Attributes {
   }
 
 private:
-  Derived* This() {
+  Derived* This() __ubsan_ignore_vptr__ {
```

Comment:
This is probably a real bug.

Reply:
Noted in https://github.com/pytorch/pytorch/issues/9055